metals/src/main/scala/scala/meta/internal/metals/Refactorings.scala

```diff
@@ -0,0 +1,175 @@
+package scala.meta.internal.metals
+
+import scala.concurrent.{Future, ExecutionContext}
+import scala.meta._
+import scala.meta.pc.CancelToken
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.mtags._
+import scala.meta.internal.semanticdb.TextDocument
+import org.eclipse.{lsp4j => l}
+
+object Refactorings {
+
+  object UseNamedArguments extends CodeAction {
```

Comment:
Can we move each refactoring into a separate source file and into the package `scala.meta.internal.codeactions`? We may end up with hundreds of code actions, which would be nice to maintain in separate source files.

Reply:
👍 I was thinking the same, makes sense! 
If only we had a “move package” code action 😅

Reply:
Done 👍